core/templates/dev/head/pages/library/library.html

```diff
@@ -90,20 +90,17 @@ <h2 class="oppia-library-h2" translate="I18N_LIBRARY_SUB_HEADER"></h2>
                translate="I18N_LIBRARY_NO_EXPLORATION_GROUPS">
           </div>
 
-          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index">
+          <div class="oppia-library-group" ng-repeat="group in libraryGroups track by $index" ng-mouseover="setActiveGroup($index)" ng-mouseout="clearActiveGroup()">
             <div class="row oppia-library-group-header-container">
-              <h2 class="oppia-library-group-header">
-                <a ng-if="group.has_full_results_page"
-                   ng-click="showFullResultsPage(group.categories, group.full_results_url)"
+              <h2 ng-class="{'oppia-library-group-header-active': activeGroupIndex === $index, 'oppia-library-group-header': activeGroupIndex != $index }">
+                <span ng-if="group.has_full_results_page"
                    translate="<[group.header_i18n_id]>">
-                </a>
+                </span>
+                <span ng-show="activeGroupIndex === $index"><i class="material-icons md-40"
+                  style="height:100%;">&#xE315;</i></span>
```

Comment:
Why do you need the style here? I suspect that this is what is causing the "jiggling" issues.